CMakeLists.txt

```diff
@@ -10,12 +42,22 @@ set(WEB3_DIR "${CMAKE_CURRENT_LIST_DIR}/webthree" CACHE PATH "The path to the we
 set(SOL_DIR "${CMAKE_CURRENT_LIST_DIR}/solidity" CACHE PATH "The path to the solidity libraries directory")
 
 # set cmake_policies
+
+testFileExists("${CMAKE_CURRENT_LIST_DIR}/webthree-helpers/cmake/EthPolicy.cmake" "Did you use 'git clone --recursive' to checkout all repositories?")
+
 include(EthPolicy)
 eth_policy()
 
 # project name and version should be set after cmake_policy CMP0048
 project(cpp-ethereum VERSION "1.2.1")
 
+# test if files exist
```

Comment:
I wouldn't bother with checking all of these individual files.     This whole "feature" is purely heuristic anyway, so if EthPolicy.cmake is there, it's safe enough to assume everything else is.

If we start checking every file then we "should" do that for every single include everywhere, which would obviously be stupid.

The heuristic check is just really to try to help the use if they forgot the --recursive.    Just the one check on the very first file should be enough for this, I think.   Thanks!    Let's chop these other ones out.